samples/nrf9160/nrf_cloud_mqtt_multi_service/Kconfig

```diff
@@ -0,0 +1,74 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menu "MQTT Multi Service Sample Settings"
+
+config POWER_SAVING_MODE_ENABLE
+	bool "Enable to request Power Saving Mode from cellular network"
+
+config LTE_INIT_RETRY_TIMEOUT_SECONDS
+	int "Sets the number of seconds between each LTE modem initialization retry"
+	default 30
+
+config CLOUD_CONNECTION_RETRY_TIMEOUT_SECONDS
+	int "If connecting to nRF Cloud takes longer than this timeout, it will be reattempted"
+	default 30
+
+config CLOUD_CONNECTION_REESTABLISH_DELAY_SECONDS
+	int "After the connection to nRF Cloud has been reset, wait this amount of time before attempting to connect again"
+	default 5
+
+config CLOUD_READY_TIMEOUT_SECONDS
+	int "If the connection to nRF Cloud does not become ready within this timeout, the sample will reset its connection and try again"
```

Comment:
Instead of having long lines after the type, some folks make this a shorter summary, and put the bulk in the help section.